src/redux/modules/todo.js

```diff
@@ -0,0 +1,17 @@
+// Actions
+const SET = 'todo/SET';
+
+// Reducer
+export default function reducer(state = {}, action = {}) {
+  switch (action.type) {
+    case SET:
+      // Perform action
+      return state;
+    default: return state;
+  }
+}
+
+// Action Creators
+export function set() {
+  return { type: SET };
+}
```

Comment:
[OPTIONAL] Kindly remove this useless file so that your repo is clean and professional.